scarlet/lite/models.py

```diff
@@ -367,6 +378,17 @@ def convolve(self, image, mode=None, grad=False):
             raise ValueError(f"mode must be either 'fft' or 'rea', got {mode}")
         return result
 
+    def render(self, model):
```

Comment:
should this pass along the other keywords to convolve such as `mode` and `grad` ?

Reply:
I don't think so, because those fields only exist in `LiteObservation`, not `Observation`, and including them in `render` would also seem to require them in `Observation.render` where they don't make sense. This is part of the reason that I changed the name of the method, because it has slightly different behavior in scarlet lite.

Reply:
OK.

I noticed that you use mode='real' in the re weighting code.  Is that the preferred method?

Reply:
Yes, because using FFTs will create small amplitude artifacts in the box that (potentially) have zero values when convolved using real convolutions.